benchmarks/test_benchmark_coo.py

```diff
@@ -29,15 +29,15 @@ def bench():
         x @ y
 
 
-def elemwise_test_name(param):
+def id_of_test(param):
```

Comment:
I'd recommend naming this `get_test_id`.

Reply:
Done